src/lang/Messages.properties

```diff
@@ -2032,6 +2032,7 @@ view.options.label.responsepanelpos.above   = Request Shown Above Response
 view.options.label.responsepanelpos.sideBySide = Panels Side by Side
 view.options.label.responsepanelpos.tabs    = Tabs Side by Side
 view.options.label.showSplashScreen			= Show splash screen
+view.options.label.showlocalconnectrequests = Persist and show (local) CONNECT requests
```

Comment:
Strictly this is correct, but might this actually confuse most users? Could just be 'Show (local) CONNECT requests' ?